src/lnav.hh

```diff
@@ -251,6 +251,7 @@ struct _lnav_data {
 
     std::list<pid_t>                        ld_children;
     std::list<piper_proc *>                 ld_pipers;
+    bool                                    ld_secure_mode;
```

Comment:
Add a flag instead of a bool field (i.e.  LNB_SECURE_MODE, LNF_SECURE_MODE), see the flags near the top of the file.